src/js/tabs/firmware_flasher.js

```diff
@@ -647,6 +647,16 @@ firmware_flasher.initialize = function (callback) {
             $('input.flash_manual_baud').prop('checked', false);
         }
 
+        $('input.classicbuild_mode').change(function () {
+            const status = $(this).is(':checked');
+
+            $('select[name="radioProtocols"').enabled = !status;
+            $('select[name="telemetryProtocols"').enabled = !status;
+            $('select[name="motorProtocols"').enabled = !status;
+            $('select[name="options"').enabled = !status;
```

Comment:
```suggestion
            $('select[name="radioProtocols"').prop("disabled", status);
            $('select[name="telemetryProtocols"').prop("disabled", status);
            $('select[name="motorProtocols"').prop("disabled", status);
            $('select[name="options"').prop("disabled", status);
```

Reply:
haha - just noticed that and corrected it :)